examples/text_input_aligned/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "text_input_aligned"
+version = "0.1.0"
+authors = ["Subroto Biswas <sub14biswas@gmail.com>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+iced = { path = "../..", features = ["async-std", "debug"] }
+log = "0.4.0"
+log4rs = "0.11.0"
```

Comment:
We use `env_logger`. We should avoid adding additional development dependencies unless necessary.